MantidQt/MantidWidgets/src/ColorBarWidget.cpp

```diff
@@ -1,13 +1,10 @@
 #include "MantidQtAPI/MantidColorMap.h"
 #include "MantidQtMantidWidgets/ColorBarWidget.h"
 #include "MantidQtAPI/QScienceSpinBox.h"
-#include "qwt_scale_div.h"
 #include "MantidQtAPI/PowerScaleEngine.h"
-#include <iosfwd>
 #include <qwt_scale_map.h>
 #include <qwt_scale_widget.h>
 #include <QKeyEvent>
-#include <qwt_plot.h>
```

Comment:
ReSharper with VS2015 grays out all unused headers, with a pretty good accuracy, at least for now.